### DISCUSSION TITLE
A ghost notification that never disappear in my github acc

### DISCUSSION THREAD

**Discussion by BakaFT** [community/community#112265] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Well I don't know where this post should be posted so I'm just posting it in General.

I have a notification, but actually there is nothing here and I just can not get rid of it, thought that was a bug.
Has somebody else encountered this issue too?

![image](https://github.com/community/community/assets/8694498/a0d98680-2515-440a-84ae-392c7b272f1d)

**Comment by ghostinhershell** [comment]
Hi @BakaFT, 

Thanks so much for writing in. Are you still running into this issue on your end? Sometimes [clearing your cache and cookies](https://kb.iu.edu/d/ahic) does the trick.

**Comment by BakaFT** [comment]
> Hi @BakaFT,
> 
> Thanks so much for writing in. Are you still running into this issue on your end? Sometimes [clearing your cache and cookies](https://kb.iu.edu/d/ahic) does the trick.

I tried to clear cookies&disable cache but it didn't work, even in `inPrivate` window after login it still exists.

I think this should be a server side issue now

**Reply by NullHypothesis** [reply]
I have the same problem. Ever since a few days ago, the UI shows the notification icon but there is no notification. Clearing cookies did not help.

**Comment by adah1972** [comment, marked as answer]
Check out this discussion: <https://github.com/orgs/community/discussions/111217>.

It solved the problem for me.

**Reply by NullHypothesis** [reply]
Worked for me too, thanks! That said, displaying a notification icon for a no-longer-existing tag is indeed a bug.

**Reply by BakaFT** [reply]
This did work for me, thanks!
